jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -203,9 +210,16 @@ node ()  {
 						echo "setting lambda execution role as $role";
 						setLambdaExecutionRole(role);
 						sendEvent(service, branch, runtime, "PRODUCTION", region, domain, role, _event, Event_Status.COMPLETED, "")
+						addCommonExcludes()
```

Comment:
If we are adding this feature,  there will be changes in https://github.com/tmobile/jazz/tree/master/serverless-config-pack